src/app/client/src/app/modules/dashboard/components/report/report.component.html

```diff
@@ -19,7 +19,7 @@
       </div>
       <div class="stackable grid m-0 sb-mid-container-min-height dashboard-body-scroll">
         <div class="sb-sideview-tab-container pb-48 p-0 w-100" *ngIf="showComponent">
-          <ng-container *ngIf="(report$ | async) as currentReport else loading">
+          <ng-container *ngIf="currentReport else loading">
```

Comment:
Please keep using async pipe for auto unsubscription purposes. Also, remove the explicit subscription from the .ts file and include your logic inside this observable.

Reply:
fixed